docs/tutorials/content.md

```diff
@@ -0,0 +1,118 @@
+## ITowns' basics
+
+### 1 - Introduction
+
+### 2 - The `View` : a support for data visualization
+
+- What is a `View` ? 
+- A `View` contains the `canvas` WebGL uses to render graphics.
+- It implements a `Camera`, which extends Three.js Camera.
+- How to interact with a `View` : the controls allows moving the camera within its `View`.
+- A short description of the default controls and the default keybindings.
+- Two specific types of `View` : `GlobeView` and `PlanarView`, with associated controls.
+- A brief example on how to implement a `GlobeView`, and its explanations :
+>```js
+>    const viewerDiv = document.getElementById('viewerDiv');
+>    const view = new itowns.GlobeView(viewerDiv);
+>```
+
+### 3 - Data visualization process
+
+- Data is displayed as layered data in iTowns (`Layer`, `ColorLayer`, `ElevationLayer`, `GeometryLayer`).
+- The process of displaying data is in three stages : fetch, parse and convert.
+
+#### Fetch, parse and convert
+
+- Data is fetched thanks to `Source`. Different types of `Source` exist.
+- Fetched data is then parsed to create iTowns intermediate between raw data and Three.js objects.
+- Parsed data is converted into Three.js compatible 3D objects.
+
+#### Create a simple parser
+
+- An example on how to create a simple parser.
+
+**_QUESTION : SHOULD THE EXAMPLE ON HOW TO CREATE A SIMPLE PARSER BE AFTER THE `Feature` MENTION ? 
+IF DONE SO, USER COULD KNOW WHAT THEY INTEND TO CREATE WITH THEIR PARSER (ASSUMING THEY PARSE SOME VECTOR DATA)._**
+
+### 5 - Vector data processing
+
+#### A) `Feature`
+
+- To display vector data, iTowns parses it to create instances of `Feature`.
```

Comment:
- features types 
- the features are display in `ColorLayer` (Style) or `GeometryLayer` (extrude, color (soon in Style too))